Assets/LeapMotion/Scripts/LeapServiceProvider.cs

```diff
@@ -220,7 +223,7 @@ public class LeapServiceProvider : LeapProvider {
       } else {
         leap_controller_.Frame(_untransformedUpdateFrame);
       }
-
+      imageTimeStamp = leap_controller_.FrameTimestamp();
```

Comment:
Why is FrameTimestamp considered to be the image timestamp?  In LeapImageRetriever we request images based on the timestamp of CurrentFrame.